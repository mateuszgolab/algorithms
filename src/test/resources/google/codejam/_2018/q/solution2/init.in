5
5
5 6 8 4 3
3
8 9 7
4
8 9 7 6
5
8 9 7 6 5
6
8 9 7 6 5 4