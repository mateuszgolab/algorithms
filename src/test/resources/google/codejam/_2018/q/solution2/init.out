Case #1: OK
Case #2: 1
Case #3: 0
Case #4: 3
Case #5: 0
